src/fiatconnect/FiatDetailsScreen.test.tsx

```diff
@@ -237,7 +248,31 @@ describe('FiatDetailsScreen', () => {
     // Should see an error message saying the account number field is invalid
     // immediately since the field loses focus
     expect(queryByTestId('errorMessage-accountNumber')).toBeTruthy()
-    expect(queryByText('fiatAccountSchema.accountNumber.errorMessage')).toBeTruthy()
+    expect(queryByText('fiatAccountSchema.accountNumber.errorMessageDigit')).toBeTruthy()
```

Comment:
super nit: it's recommended to use `getBy*` if the thing should exist, and `queryBy*` if expecting the thing not to exist. the `getBy*` methods give better errors when it fails :) https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence

Reply:
sounds good, I'll address this in a follow up since every test in this file needs fixing.